scripts/apps/web-publisher/styles/_helpers_variables.sass

```diff
@@ -0,0 +1,45 @@
+$superdesk-green:       rgba(49,172,102,1);
```

Comment:
Is web publisher branding different from Superdesk? If not, let's use the variables already available in [`~variables/_colors.scss`](https://github.com/superdesk/superdesk-client-core/blob/master/styles/sass/variables/_colors.scss). I can see that the `$superdesk-green` is there, although slightly different.